namer/consul/src/test/scala/io/buoyant/namer/consul/ConsulTest.scala

```diff
@@ -20,13 +20,13 @@ class ConsulTest extends FunSuite {
 
   test("parse config") {
     val yaml = s"""
-                    |kind: io.l5d.experimental.consul
+                    |kind: io.l5d.consul
                     |host: consul.site.biz
                     |port: 8600
       """.stripMargin
 
     val mapper = Parser.objectMapper(yaml, Iterable(Seq(ConsulInitializer)))
-    val consul = mapper.readValue[NamerConfig](yaml).asInstanceOf[consul]
+    val consul = mapper.readValue[NamerConfig](yaml).asInstanceOf[ConsulConfig]
     assert(consul.host == Some("consul.site.biz"))
     assert(consul.port == Some(Port(8600)))
   }
```

Comment:
Maybe add the experimental flag to the yaml, and `assert(!consul.disabled)` -- or alternatively leave the yaml as is and `assert(consul.disabled)`?